berrylan.desktop.in

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+_Name=berrylan
+Exec=qmlscene %U main.qml
```

Comment:
This won't work. You'll need to run the berrylan binary as that exports the C++ classes into QML.

Reply:
how do I run the berrylan binary?

Reply:
just running `berrylan`, (assuming it's installed into <click-package-root>/bin or similar.